openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -163,7 +167,7 @@ jQuery(function () {
 
     // conditionally load real time signup functionality based on class in the page
     if (document.getElementsByClassName('olform create validate').length) {
-        import('./realtime_account_validation.js')
+        import(/* webpackChunkName: "rt-validation" */'./realtime_account_validation.js')
```

Comment:
Let's keep these names identical; makes it easier to find chunks.
```suggestion
        import(/* webpackChunkName: "realtime-account-validation" */'./realtime_account_validation.js')
```